src/dotnet/project.json

```diff
@@ -26,6 +26,8 @@
     }
   },
   "dependencies": {
+    "NuGet.CommandLine.XPlat": "3.6.0-beta.1.msbuild.1",
```

Comment:
Were you able to remove these?  I still can't see where they are necessary.

---
In reply to: [74501423](https://github.com/dotnet/cli/pull/4035#discussion_r74501423) [](ancestors = 74501423)